src/components/UserDisplay.vue

```diff
@@ -19,20 +19,36 @@ export default {
   name: 'UserDisplay',          
     data() {
         return{
-        user: this.$store.state.user.currentUser,
+        user: {}
         }
     },
-  methods: {
-    getName(){
-      console.log(this.user)
-      if(!this.user.firstName || !this.user.lastName || this.user.firstName === "" || this.user.lastName === ""){
-        return "Placeholder Holdersen Place"
-      }
-      else{
-        return this.user.firstName + " " +  this.user.lastName
-      }
+  methods: {     
+    
+    
+     //getting the name from the user and making sure there is something in it
+    getName() {
+          this.user = this.$store.state.user.currentUser
```

Comment:
Make a 'created' function, retrieving and saving user data and change method accordingly.